tests/test_indices.py

```diff
@@ -463,6 +473,16 @@ def test_run_start_at_0(self, pr_series):
         assert out[0] == 10
 
 
+class TestMaximumConsecutiveSummerDays:
+    def test_simple(self, tasmax_series):
+        a = np.zeros(365) + 273.15
+        a[5:15] += 30
+        tx = tasmax_series(a, start="1/1/2010")
+        out = xci.maximum_consecutive_summer_days(tx, thresh="25 C", freq="M")
```

Comment:
Didn't we change the name of this recently?

Reply:
No, it's just that I did the initial checkout on branch fix-61, not master. Once we merge fix-61, that should go away.